README.md

```diff
@@ -37,6 +38,18 @@
 
 #### 2. Activate Plugin
 
+## Set Up ClassifAI account
+
+ClassifAI is a sophisticated solution that we want organizations of all shapes and sizes to count on. To keep adopters apprised of major updates and beta testing opportunities, gather feedback, and prioritize common use cases, we're asking for a little bit of information in exchange for a free key. Your information will be kept confidential.
+
+#### 1. Sign up for a ClassifAI account
+- Register for a free ClassifAI account [here](https://classifaiplugin.com/#cta).
+- Check for an email from ClassifAI Team which contains the registration key.
```

Comment:
Let's update to `ClassifAI Team` and note the email address is opensource@10up.com so they can whitelist for that in their email client.